src/data/community-events.json

```
@@ -187,5 +187,14 @@
     "description": "ETHIndia is a hackathon — a movement with the goal of inspiring and fostering a community of builders promoting technological advancements for Ethereum.",
     "startDate": "2023-12-8",
     "endDate": "2023-12-10"
+  },
+  {
+    "title": "DEVCON 7 - Southeast Asia",
```

```suggestion
    "title": "Devcon 7 - Southeast Asia",
```

Just changing to proper capitalization based on the site.